pkg/deployments/package.json

```diff
@@ -12,17 +12,19 @@
   "bugs": {
     "url": "https://github.com/balancer-labs/balancer-v2-monorepo/issues"
   },
+  "main": "dist/index.js",
+  "module": "dist/index.esm.js",
+  "browser": "dist/index.umd.js",
+  "typings": "dist/index.d.ts",
   "files": [
-    "index.ts",
-    "tasks/**/abi/*",
-    "tasks/**/output/*",
-    "!test.json"
+    "dist/"
   ],
   "scripts": {
     "lint": "eslint . --ext .ts --max-warnings 0",
     "test": "hardhat test ./**/test/*.ts",
     "test:deployed": "yarn test --network mainnet",
-    "test:fork": "yarn test --fork mainnet --block-number 12731159"
+    "test:fork": "yarn test --fork mainnet --block-number 12731159",
+    "prepublishOnly": "tsc"
```

Comment:
Let's move this to `build` and run it on CI. Also, it seems to be currently failing:

```
error TS2339: Property 'ethers' does not exist on type '{ default: HardhatRuntimeEnvironment; config: HardhatConfig; hardhatArguments: HardhatArguments; tasks: TasksMap; run: RunTaskFunction; network: Network; artifacts: Artifacts; }'.

17   const { ethers } = await import('hardhat');
```